[2024-08-27T01:25:02.087+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_postgresql_etl.load scheduled__2024-08-26T01:16:49.954923+00:00 [queued]>
[2024-08-27T01:25:02.102+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_postgresql_etl.load scheduled__2024-08-26T01:16:49.954923+00:00 [queued]>
[2024-08-27T01:25:02.103+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2024-08-27T01:25:02.126+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load> on 2024-08-26 01:16:49.954923+00:00
[2024-08-27T01:25:02.137+0000] {standard_task_runner.py:57} INFO - Started process 466 to run task
[2024-08-27T01:25:02.143+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_postgresql_etl', 'load', 'scheduled__2024-08-26T01:16:49.954923+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp9vn4zjpg']
[2024-08-27T01:25:02.146+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask load
[2024-08-27T01:25:02.225+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_to_postgresql_etl.load scheduled__2024-08-26T01:16:49.954923+00:00 [running]> on host 41675a30b3a8
[2024-08-27T01:25:02.399+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_postgresql_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-08-26T01:16:49.954923+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-26T01:16:49.954923+00:00'
[2024-08-27T01:25:02.480+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-08-27T01:25:02.506+0000] {logging_mixin.py:150} INFO - Table 'transformed_data6' created or already exists.
[2024-08-27T01:25:02.997+0000] {logging_mixin.py:150} INFO - Inserted 1000 rows into 'transformed_data6'
[2024-08-27T01:25:02.999+0000] {logging_mixin.py:150} INFO - Sample of uploaded data:
[2024-08-27T01:25:02.999+0000] {logging_mixin.py:150} INFO - (1, 19, 27.9, '18-29', 'Overweight')
[2024-08-27T01:25:03.000+0000] {logging_mixin.py:150} INFO - (2, 18, 33.77, '18-29', 'Obese')
[2024-08-27T01:25:03.000+0000] {logging_mixin.py:150} INFO - (3, 28, 33.0, '18-29', 'Obese')
[2024-08-27T01:25:03.000+0000] {logging_mixin.py:150} INFO - (4, 33, 22.705, '30-44', 'Normal')
[2024-08-27T01:25:03.001+0000] {logging_mixin.py:150} INFO - (5, 32, 28.88, '30-44', 'Overweight')
[2024-08-27T01:25:03.002+0000] {logging_mixin.py:150} INFO - Total rows in transformed_data6: 2000
[2024-08-27T01:25:03.003+0000] {logging_mixin.py:150} INFO - Data successfully loaded into PostgreSQL transformed_data6 table
[2024-08-27T01:25:03.004+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-27T01:25:03.018+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=mysql_to_postgresql_etl, task_id=load, execution_date=20240826T011649, start_date=20240827T012502, end_date=20240827T012503
[2024-08-27T01:25:03.067+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-27T01:25:03.095+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
